docs/dev/fundamentals/faq.md

```diff
@@ -0,0 +1,214 @@
+
+Its architecture is based on the following components:
+
+- zkVM, a Turing-complete RISC-like virtual machine optimized for proving in a ZKP circuit. It has several different implementations:
+    - Executor: fast native execution on CPU.
+    - Witness generator: native executor to generate ZKP witness.
+    - Prover: the actual ZKP circuit implementation.
+- LLVM-based compiler with:
+    - Solidity frontend (more precisely: Yul frontend).
+    - Vyper frontend.
+    - zkVM backend.
+- Special-purpose circuits (heavily relying on PLONK’s custom gates and lookup tables) as “precompiles” for computationally intensive operations, such as:
+    - Non-algebraic hashes (Keccak, SHA256, Blake2).
+    - Storage access (Merkle paths).
+    - Elliptic curve pairing.
+- Recursive aggregation circuit (combines the proofs from the above-mentioned parts).
+
+### zkEVM vs EVM
+
+Apart from the opcodes and gas metering disparity, zkVM strictly inherits the EVM programming model and its invariants, including the ABI calling conventions. One important thing to emphasize is that the zkVM supports rollbacks and provably revertible transactions. It guarantees mutual protection: users can not stall the network with bombardment by revertible transactions, and the escape hatch (priority queue) protect the user’s ability to include any transactions into the blocks.
+
+As a result, developers can fully rely on the censorship-resistance provided by L1 without having to introduce any changes related to the escape-hatch mechanism. This means that assets in a zkRollup account on zkSync will have exactly the same security guarantees as on L1.
+### EVM Improvements
+
+While maintaining maximum compatibility, the zkEVM has significant improvements over the EVM that increase adoption and benefit our ecosystem partners.
+
+- **Our compiler is based on LLVM**. LLVM-based compilers (Low-Level Virtual Machine) have become the default compiler for Mac OS X, iOS, FreeBSD, and Android systems and are among the most widely used because they:
+    - Enable us to improve the efficiency over the original EVM bytecode because with LLVM we can take advantage of the many optimizations and tools available in this mature ecosystem.
+    - Pave the way for us to add support for integrating codebases written in other programming languages with LLVM frontend. By doing so, developers can build dApps and use blockchains in ways that are currently not possible.
+- **Account Abstraction is included in our zkEVM**. This is a long-anticipated feature in the Ethereum dev community which improves developer adoption and user experience in a number of ways:
+    - Native support for smart contracts wallets (like Argent), which is critical for onboarding mainstream users.
+    - Much better UX for multisigs.
+    - Transaction fees can be paid in any token.
```

Comment:
Transaction fees can be paid in any token + "using paymasters" with a link to the paymasters section of the docs.